Remove extra space for Nathan Lin

nlin24 opened the pull request:
Removed extra space for Nathan Lin entry

B-AlexiaYoung committed:
Add Beryl Young to Contributors list

nlin24 committed:
Remove extra space for Nathan Lin

nlin24 committed:
Remove extra space for Nathan Lin

Commit:
Add Humbl3Stud3nt

sara-02 reviewed (commented):
@nlin24 Thanks for this change, there are other names in the file that require similar changes.Can you update the PR for those names as well?
Also, rebase your branch to get the list of latest. contributors.

dmagee15 committed:
added-dmagee15

nnguyen74 committed:
Add Nguyen Nguyen to Contributors list

bauerpm7 committed:
Add Michael Bauer to Contributors list

HarshBhardwaj committed:
Add Harsh Bhardwaj to Contributors list

nlin24 committed:
Remove extra spaces in Contributor.md

nlin24 committed:
Remove extra spaces in Contributor.md

jeffreyliu34 committed:
Add Jeffrey Liu to Contributors list

nlin24 committed:
Merge branch 'master' into removeSpace

nlin24 commented:
@sara-02 Hi, thanks for your feedback. I removed the extra spaces I could found. Can you please review again and provide feedback? Thanks a bunch.

sara-02 commented on Contributors.md:
```diff
@@ -2025,12 +2025,5 @@
 - [Cristian Rodriguez](https://github.com/crisr10)
-- [Aman Jain] (https://github.com/jainaman1398)
-- [Nathan Lin] (https://github.com/nlin24)
-- [Humbl3Stud3nt] (https://github.com/Humbl3Stud3nt)
```
@nlin24 Any reason for removing these contributors? Have they been accommodated above, or was this by mistake? Please take a look at it.

nlin24 replied:
@sara-02 Hi, thanks for the feedback. That was a mistake. Sorry.

nlin24 replied:
@sara-02 I correct the missing contributors from line 2028-2030 and committed again. It was weird because my branch was only missing Humbl3Stud3nt from line 2030. I am not sure why is that. Please let me know if you see these three names have been added back. Thank you.

nlin24 committed:
Add back missing contributors

Add back Aman Jain, Nathan Lin, and Humbl3Stud3nt, which were deleted by accident.

nlin24 committed:
Add back missing contributors

Add back Aman Jain, Nathan Lin, and Humbl3Stud3nt, which were deleted by accident.

sara-02 commented:
@nlin24 If you rebase from latest master it should fix the issue. Please trying doing that :)

nlin24 committed:
Abort rebase

nlin24 committed:
Abort rebase

nlin24 committed:
Merge branch 'removeSpace' of https://github.com/nlin24/first-contributions into removeSpace

nlin24 committed:
Remove the extra brackets [] by mistakes

nlin24 committed:
Merge branch 'master' into removeSpace
